CONTRIBUTING.md

```diff
@@ -1,7 +1,57 @@
-# How to Contribute
+# Contributing
 
-We'd love to accept your patches and contributions to this project. There are
-just a few small guidelines you need to follow.
+We'd love to accept your patches and contributions to this project.
+
+## Development Environment
+
+If you are contributing documentation (rather than samples) and want to be able
+to view it in your browser, the easiest way to do so is to run the provided
+development server.
+
+We use [GitHub Pages][1] to make this documentation available, which uses
+[Jekyll][2] under the hood.
+
+If you have [Docker][3] installed, simply clone this repository and run the
+`serve.sh` file at the root of the repository. This script does two things:
+
+  * It builds the provided Docker image (unless you already have it) and tags
+    it as `googleapis-site`.
+  * It runs the `googleapis-site` image.
+
+The Jekyll development server uses port 4000 by default, so simply point your
```

Comment:
I think we can delete this word with no loss of meaning.

Reply:
Done.